src/main/java/com/fabriccommunity/spookytime/mixin/EndermanPumpkinMixin.java

```diff
@@ -0,0 +1,34 @@
+package com.fabriccommunity.spookytime.mixin;
+
+import com.fabriccommunity.spookytime.MixinHelpers;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.entity.EntityData;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.SpawnType;
+import net.minecraft.entity.mob.EndermanEntity;
+import net.minecraft.entity.mob.HostileEntity;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.LocalDifficulty;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+/**
+ * @author Indigo Amann
+ */
+@Mixin(EndermanEntity.class)
+public abstract class EndermanPumpkinMixin extends HostileEntity {
+    @Shadow public abstract void setCarriedBlock(BlockState blockState_1);
+
+    protected EndermanPumpkinMixin(EntityType<? extends HostileEntity> entityType_1, World world_1) {
+        super(entityType_1, world_1);
+    }
+
+    @Override
```

Comment:
`@Override` is evil for mixins-- could you change it to an `@Inject` at the head instead? The if-statement line is also a bit confusing; would be nice to have that spaced out with braces (double ternary is a bit too confusing).

Reply:
`@Override` and `@Overwrite` are different things in mixins. The first one overrides a superclass method in a subclass mixin and the second one replaces a method in the target class.

Reply:
I think it will, but you can't really inject to superclass methods in subclasses. Not sure what's the best way to handle this.

Reply:
Perhaps you change the target to the super and check instanceof Enderman? Might work better.